spec/requests/secure_cookies_spec.rb

```diff
@@ -4,28 +4,30 @@
   context 'with plain HTTP' do
     it 'flags all cookies sent by the application as HttpOnly and SameSite=Lax' do
       get root_url
-      cookie_count = response.headers['Set-Cookie'].split("\n").count
+      cookies = response.headers['Set-Cookie']
+      cookie_count = cookies.count
 
-      expect(response.headers['Set-Cookie']).to_not include('; Secure')
-      expect(response.headers['Set-Cookie'].scan('; HttpOnly').size).to eq(cookie_count)
-      expect(response.headers['Set-Cookie'].scan('; SameSite=Lax').size).to eq(cookie_count)
+      expect(cookies.count { |x| x.match?(SecureCookies::SECURE_REGEX) }).to eq(0)
```

Comment:
maybe a good use for `.any?` or `.none?`

```suggestion
      expect(cookies.any? { |x| x.match?(SecureCookies::SECURE_REGEX) }).to eq(false)
```

Reply:
Thanks, changed in be252453fd2fc941c6e3d1f2579ded427ebdb8f1